cl/assets/static-global/css/override.css

```diff
@@ -1352,11 +1352,36 @@ emphasis {
   margin-right: 0.25em;
 }
 
-
 [data-loading] {
   display: none;
 }
 
+.webhooks textarea {
+  resize: vertical;
+}
```

Comment:
I don't think we have many textareas. Could you grep the code and see how many we have? I can't think of anywhere that we wouldn't want a `resize: vertical`. If that's true, let's just make this part of our CSS for all text areas.

Reply:
Yeah, I looked for textareas and we only use them when saving a favorite, making visualization, within the contact form, and here. So it's ok to use `resize: vertical ` globally.